docs/ossm/README.md

```diff
@@ -0,0 +1,196 @@
+[Return to Project Root](../../)
+
+# Table of Contents
+
+- [User Documentation](#user-documentation)
+- [istio-csr Integration](#istio-csr-integration)
+  - [Common Setup](#common-setup)
+  - [InPlace Strategy](#inplace-strategy)
+  - [RevisionBased Strategy](#revisionbased-strategy)
+  - [Verify](#verify)
+  - [RevisionBased Upgrades](#upgrades---revisionbased)
+
+# User Documentation
+
+This documentation is specific to the OpenShift Service Mesh product and may differ slightly from the [Sail Operator documentation](../README.md). If you are using the OpenShift Service Mesh 3 operator, you should consider this documentation the source of truth. If there is not documentation here for a particular topic, follow the [Sail Operator documentation](../README.md).
+
+# istio-csr Integration
+
+Configuring your `Istio` resource to use [istio-csr](https://github.com/cert-manager/istio-csr/tree/main) largely follows the [cert-manager documentation](https://cert-manager.io/docs/usage/istio-csr/) but you will need to adjust a few settings based on which `updateStrategy` you are using.
+
+## Common Setup
+
+These steps are the same for each `updateStrategy`.
+
+1. Install the [cert-manager Operator for Red Hat OpenShift](https://docs.redhat.com/en/documentation/openshift_container_platform/4.10/html/security_and_compliance/cert-manager-operator-for-red-hat-openshift#cert-manager-install-console_cert-manager-operator-install).
```

Comment:
The linked document points to OCP 4.10 version. We should probably link the latest 4.16 doc here 
https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/security_and_compliance/cert-manager-operator-for-red-hat-openshift#cert-manager-operator-about

Reply:
Ah yes, I probably just copied whatever url I was looking at without paying much attention to the version.